packages/core/schematics/migrations/control-flow-migration/types.ts

```diff
@@ -116,18 +116,7 @@ export interface AliasAttributes {
   aliases: Map<string, string>;
 }
 
-export interface ParseResult {
-  tree: ParseTreeResult | undefined;
-  errors: MigrateError[];
-}
-
-/**
- * Represents an error that happened during migration
- */
-export type MigrateError = {
-  type: string;
-  error: unknown;
-};
+export {MigrateError} from '../../utils/parse_html';
```

Comment:
```suggestion
export type {MigrateError} from '../../utils/parse_html';
```
Because of the `isolatedModules` option, we need that to be an `export type`